sources/packages/api/src/database/entities/student.model.ts

```diff
@@ -77,4 +77,11 @@ export class Student extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   sinValidation: SINValidation;
+  /**
+   * Mandatory sin consent required by user when creating a new profile.
```

Comment:
The SIN must be uppercase.
As much as possible we can make the comments be the same between the model and the DB.